core/src/kura.rs

```diff
@@ -339,176 +337,164 @@ impl Kura {
     }
 }
 
-/// The interface for the **block store**, which is where [Kura],
-/// the block storage subsystem, stores its blocks.
-///
-/// The [`BlockStoreTrait`] defines and implements functionality used by Kura
-/// for every platform. The default implementation is `StdFileBlockStore`,
-/// which uses `std::fs`.
-pub trait BlockStoreTrait: Debug {
-    /// Read a series of block indices from the block index file and
-    /// attempt to fill all of `dest_buffer`.
-    ///
-    /// # Errors
-    /// IO Error.
-    fn read_block_indices(
-        &self,
-        start_block_height: u64,
-        dest_buffer: &mut [(u64, u64)],
-    ) -> Result<()>;
+// Marker trait for typestate. Not sealed, since while it
+// doesn't need to be implemented by a user, implementing it
+// shouldn't lead to problematic behaviour.
```

Comment:
```suggestion
/// Marker trait for typestate. Not sealed, because while it
/// doesn't need to be implemented by a user, implementing it
/// shouldn't lead to problematic behaviour.
```

Reply:
It's more of implementation detail, don't think we need to clutter the docs with it. Basically if you need to touch it, you are already changing this code directly.

Reply:
ok. it's just the similar comments just below this one are doc comments, I thought this might've been a typo